File: solutions/3223/solution.py
class Solution:
    def minimumLength(self, s: str) -> int:
        alpha = [0] * 26

        for ch in s:
            index = ord(ch) - ord('a')
            alpha[index] = 1 if alpha[index]%2 == 0 else 2
        
        return sum(alpha)

File: solutions/3223/solution.cpp
class Solution {
public:
    int minimumLength(string s) {
        vector<int> alpha(26,0);
        for(char ch : s){
            alpha[ch -'a'] = (alpha[ch-'a']%2 == 1) ? 2 : 1;
        }
        int count = 0;
        for(int i : alpha) count+=i;
        return count;
    }
};